packages/sui-react-initial-props/package.json

```diff
@@ -4,7 +4,7 @@
   "main": "lib/index.js",
```

Comment:
Should [typings](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package) field be added here ? For example:
```
{
  "typings": "dist/index.d.ts",
}
```